packages/vats/src/core/basic-behaviors.js

```diff
@@ -207,6 +207,11 @@ export const makeAddressNameHubs = async ({
 };
 harden(makeAddressNameHubs);
 
+const AccountFlags = /** @type {const} */ ({
```

Comment:
from discussion, these are `PowerFlags`. best to keep the name consistent. though it's worth a comment here that this enum is a subset of all known power flags.